tasks/spellcheck.rb

```diff
@@ -17,17 +17,31 @@
 
 namespace :spellcheck do
   task :fetch_common do
-    sh "wget https://raw.githubusercontent.com/chef/chef_dictionary/master/chef.txt -O chef_dictionary.txt"
+    sh "wget -q https://raw.githubusercontent.com/chef/chef_dictionary/master/chef.txt -O chef_dictionary.txt"
```

Comment:
Does this file need to go in `.gitignore`?

Reply:
It's already in there